buildIPFS/json/1

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Your Collection #1",
+  "description": "Remember to replace this description",
+  "image": "ipfs://QmSbPaPQqbHYEs4x5f3rsyxbHBFTYLVf8qJd9HqAuywbyK/1.png",
+  "dna": "6e82676d9271148253fed8d8509dcbd7191aed93",
+  "edition": 1,
+  "date": 1650565669844,
+  "attributes": [
```

Comment:
seems like the metadata attributes have nothing common with token images ? :)

Reply:
for simplicity you can just have name, description and image properties in token metadata.

Reply:
There are additional attributes that allow ERC721, and they're shown in the NFT Properties tab, It's just for fun)
![Screen Shot 2022-05-08 at 19 39 44](https://user-images.githubusercontent.com/77342893/167303820-9bca1a36-ea15-4d23-87b3-de61920eda37.png)
)

https://testnets.opensea.io/assets/0x1bfa9c04ed5ec1b6b435d1ce3ef08fdebeab879f/1